docs/server/source/cloud-deployment-templates/node-on-kubernetes.rst

```diff
@@ -67,25 +75,30 @@ the PersistentVolumeClaim would get stuck in a "Pending" state.
 For future reference, the command to create a storage account is
 `az storage account create <https://docs.microsoft.com/en-us/cli/azure/storage/account#create>`_.
 
-Create a Kubernetes Storage Class named ``slow``
-by writing a file named ``azureStorageClass.yml`` containing:
 
-.. code:: yaml
+MongoDB exports two volume mounts with correct permissions from inside the
+container to the outside world. We will accordingly create two storage classes
+and persistent volume claims in kubernetes.
+
+Get the files ``mongo-data-db-sc.yaml`` and ``mongo-data-configdb-sc.yaml``
+from GitHub using:
 
-   kind: StorageClass
-   apiVersion: storage.k8s.io/v1beta1
-   metadata:
-     name: slow
-   provisioner: kubernetes.io/azure-disk
-   parameters:
-     skuName: Standard_LRS
-     location: <region where your cluster is located>
+.. code:: bash
 
-and then:
+   $ wget https://raw.githubusercontent.com/bigchaindb/bigchaindb/master/k8s/mongodb/mongo-data-db-sc.yaml
+   $ wget https://raw.githubusercontent.com/bigchaindb/bigchaindb/master/k8s/mongodb/mongo-data-configdb-sc.yaml
```

Comment:
Can we do that as a part of the next PR. I had thought about it, but it makes sense to do it step by step at the first iteration and then make it shorter?